craft_providers/lxd/lxd_instance.py

```diff
@@ -660,3 +660,17 @@ def is_pro_enabled(self) -> bool:
             project=self.project,
             remote=self.remote,
         )
+
+    def enable_pro_service(self, service: str) -> None:
```

Comment:
should probably be a `list[str]`, no?

Reply:
The corresponding Pro API endpoint allows only for one service at a time.

Reply:
right but if someone wants to enable multiple services, *some code* is going to have to loop and call the api multiple times, and it might as well be this api, no?

Reply:
That will be handled on craft-application's side, because the list of services are passed there via `--pro` argument, so craft-application can call craft_providers.lxd.lxd_instance.enable_pro_service multiple times in a loop.

Reply:
I agree with Tiago but with a slight change because I love abstractions:

```suggestion
    def enable_pro_services(self, services: Iterable[str]) -> None:
```

Reply:
Changed the function signature so it accepts an iterable of `services` instead of a single `service`.